webui/react/src/types.ts

```diff
@@ -147,6 +147,13 @@ export interface Command {
 // TODO compelete the config object as we start using different attributes.
 export interface ExperimentConfig {
```

Comment:
**though:** it might be time to create a `types` directory and start organizing TS types...